tests/test_scylla_configure.py

```diff
@@ -162,6 +178,33 @@ def test_postconfig_script_with_bad_exit_code(self):
         with self.assertRaises(expected_exception=SystemExit):
             self.configurator.run_post_configuration_script()
 
+    def test_multipart_user_data_params_are_set(self):
+        ip_to_set = "172.16.16.84"
+
+        scylla_user_data = dict(
+                scylla_yaml=dict(
+                    cluster_name=self.test_cluster_name,
+                    listen_address=ip_to_set,
+                    broadcast_rpc_address=ip_to_set,
+                    seed_provider=[{
+                        "class_name": "org.apache.cassandra.locator.SimpleSeedProvider",
+                        "parameters": [{"seeds": ip_to_set}]}],
+                )
+            )
+        for data_type in ['json', 'yaml']:
```

Comment:
Same here. Please use tuple instead of lists for collections which are not expected to be changed as mentioned above.